app/models/tweet_prep.rb

```diff
@@ -0,0 +1,28 @@
+class TweetPrep
+  def self.bounty_hashtags(bounty)
+    QueryMarks.new.legible_mark_vector(bounty.mark_vector).sort_by{|_, b| -b}.map{|a, _| a}.take(4)
+  end
+
+  def self.worthy_bounties(n)
+    bounty_suggestion_fraction = 0.5
+    sorted_top_bountys = bounties_from_top_products.select(&:earnable_coins_cache).sort_by{|a| -a.earnable_coins_cache}
```

Comment:
Line is too long. [119/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.